sys/auto_init/auto_init.c

```diff
@@ -19,288 +19,227 @@
  * @author  Martine S. Lenders <m.lenders@fu-berlin.de>
  * @}
  */
+#define ENABLE_DEBUG    0
```

Comment:
there is no debug left in `auto_init.c` nor a `debug.h` include

Reply:
Its included in the `auto_init_macros.h` so that a debug macro can be defined there. If this header is only included in isolated compilation units then it should be fine. But I first reacted to this as well, can you think of another approach?

Reply:
Maybe move the debug message of every module to its auto-init function?
The `static inline` wrapper function was the low effort solution I could think of to not just delete the debug message of every module. And I guess we don´t want to store a `const char* dbg_msg` for every module.